test/Microsoft.AspNet.Mvc.IntegrationTests/ByteArrayModelBinderIntegrationTest.cs

```diff
@@ -92,8 +92,9 @@ private class Person
                 ParameterType = typeof(byte[])
             };
 
-            // No data is passed.
-            var operationContext = ModelBindingTestHelper.GetOperationBindingContext(httpContext => { });
+            // No data is passed. The ByteArrayModelBinder should ensure no later binders run.
+            var operationContext = ModelBindingTestHelper.GetOperationBindingContext(
+                updateOptions: ModelBindingTestHelper.UpdateOptionsToEnsureNothingFollows<ByteArrayModelBinder>);
             var modelState = new ModelStateDictionary();
```

Comment:
The method is in the PR this is based on -- #2625. The purpose of the method is to confirm everything works as-expected, not to change the overall operation. Have a look at the other PR.